### DISCUSSION TITLE
[DOODxGALLIKA] [DOODxKURRYUP] Moyen de paiement non ajouté automatiquement

### DISCUSSION THREAD

**Discussion by amel-nhar** [PayGreen/documentation#34] [Q&A] [answered]
Bonjour,

Je me permets de vous contacter suite à la création de nombreuses boutiques pour le compte de Gallika. Je voulais m'assurer du bon fonctionnement de votre plateforme car j'ai remarqué des comportements différents. 
Depuis notre plateforme Dood, j'ai créer l'intégration avec paygreen pour l'ensemble des établissements de Gallika. Ensuite je suis connectée à Paygreen pour voir si les moyens de paiement Conecs, Restoflash et Swile avaient bien été crées. 
Pour certains des établissements, les 3 moyens de paiement cités precedemment ont été crées, pour d'autres j'ai du le faire manuellement. Quel est comportement attendu ? Je n'ai pas modifié Gallika Boétie (sh_cfa1ffba059140b9aeefec8b513a0fdd) pour exemple.

De meme, pour les établissement Kurry Up, l'exemple est KURRY UP RENNEQUIN (sh_941c124bf8454f50b0e41843f5793f45).

Je vous remercie par avance pour votre retour. 
Cordialement.
Amel N'HAR chez Dood

**Comment by pielrx** [comment, marked as answer]
Bonjour,

Votre demande concerne un client en particulier.
Pourriez-vous effectuer votre demande via notre portail support client.

Bonne journée,

**Comment by amel-nhar** [comment]
Bonjour,

Puis-je avoir le lien de ce portail support client ?

Merci

**Reply by pielrx** [reply]
Bonjour,
Vous pouvez acceder au support client depuis votre back-office PayGreen (en bas à gauche de l'écran).
![image](https://github.com/user-attachments/assets/cf18fa6a-beaf-40e4-a784-47b8cf836194)
Bonne journée
